validator_manager/src/import_validators.rs

```diff
@@ -127,13 +227,31 @@ async fn run<'a>(config: ImportConfig) -> Result<(), String> {
         .create(false)
         .open(&validators_file_path)
         .map_err(|e| format!("Unable to open {:?}: {:?}", validators_file_path, e))?;
-    let validators: Vec<ValidatorSpecification> = serde_json::from_reader(&validators_file)
-        .map_err(|e| {
+
+    let validators: Vec<ValidatorSpecification> = if standard_format {
+        vec![ValidatorSpecification {
+            voting_keystore: KeystoreJsonStr(
+                Keystore::from_json_file(&validators_file_path).map_err(|e| format!("{e:?}"))?,
+            ),
+            voting_keystore_password: password.ok_or_else(|| {
+                "The --password flag is required to supply the keystore password".to_string()
```

Comment:
I have to put `ok_or_else` here to handle the `None` case (when `--password` flag) is not provided. We already have [requires here](https://github.com/sigp/lighthouse/pull/6261/files#diff-e54fe894fb02bdd0c2175fbb4d7cc6d04fa7c8b460b30e9512edb814f569f97eR154), so not sure about this here,  or under what condition will this manifest?

Reply:
Because we want to support both formats, we have to have `password` be an `Option`. Then since we want to build the `ImportConfig` before we know which format we are using, we are stuck with password being an `Option`.

The `or_else` should never trigger since the `require` will trigger before we even get here, but this pattern is fine as is imo. We have redundant patterns like this in plenty of places.

Reply:
Thank you for the explanation